internal/config/config.go

```diff
@@ -81,10 +82,35 @@ type MTLS struct {
 	TLSSecretNamespace string `mapstructure:"secretNamespace"`
 }
 
-func LoadConfig(filePath string, fileName string) (*Config, error) {
+func LoadConfig() (*Config, error) {
 	v := viper.New()
 
-	// We need to assure that defaults have been set in order to bind env variables.
+	pflag.StringP("config-name", "c", FileName, "filename of the configuration to look for")
+	pflag.StringP("config-path", "p", FilePath, "path where the configuration file is")
```

Comment:
These seem kind of redundant to me. I'd expect to either look for a hardcoded name (`nri-kubernetes.yaml`) in a harcoded list of paths (`.`, `/etc/newrelic-infra`), or to just point to the full path directly.

Reply:
It makes sense to have them divided since Viper is asking for those separately and therefore we would need to split it, no?